go/vt/topo/consultopo/server.go

```diff
@@ -37,13 +40,32 @@ import (
 )
 
 var (
-	consulAuthClientStaticFile = flag.String("consul_auth_static_file", "", "JSON File to read the topos/tokens from.")
-	// serfHealth is the default check from consul
-	consulLockSessionChecks = flag.String("topo_consul_lock_session_checks", "serfHealth", "List of checks for consul session.")
-	consulLockSessionTTL    = flag.String("topo_consul_lock_session_ttl", "", "TTL for consul session.")
-	consulLockDelay         = flag.Duration("topo_consul_lock_delay", 15*time.Second, "LockDelay for consul session.")
+	// flag vars
+	consulAuthClientStaticFile = ""
+	consulLockSessionChecks    = "serfHealth"
+	consulLockSessionTTL       = ""
+	consulLockDelay            = 15 * time.Second
 )
 
+func init() {
+	servenv.OnParseFor("vtbackup", registerConsulFlags)
+	servenv.OnParseFor("vtcombo", registerConsulFlags)
+	servenv.OnParseFor("vtctl", registerConsulFlags)
+	servenv.OnParseFor("vtctld", registerConsulFlags)
+	servenv.OnParseFor("vtgate", registerConsulFlags)
+	servenv.OnParseFor("vtgr", registerConsulFlags)
+	servenv.OnParseFor("vttablet", registerConsulFlags)
+	servenv.OnParseFor("vttablet", registerConsulFlags)
+}
+
+func registerConsulFlags(fs *pflag.FlagSet) {
```

Comment:
i'd go with `registerServerFlags` (_every_ flag in this package is a consul flag, so this is a bit confusing)